LogosLinuxInstaller.py

```diff
@@ -5,74 +5,156 @@
+        help='specify a custom wine binary path',
+    )
+    cfg.add_argument(
+        '-L', '--delete-log', action='store_true',
+        help='delete the log file',
+    )
+    cfg.add_argument(
+        '-P', '--passive', action='store_true',
+        help='run product installer non-interactively',
+    )
+
+    # Define runtime actions (mutually exclusive).
+    grp = parser.add_argument_group(
+        title="subcommands",
+        description="these options run specific subcommands; only 1 at a time is accepted",
+    )
+    cmd = grp.add_mutually_exclusive_group()
+    cmd.add_argument(
+        '--install-app', action='store_true',
+        help='install FaithLife app',
+    )
+    cmd.add_argument(
+        '--run-installed-app', '-C', action='store_true',
+        help='run installed FaithLife app',
+    )
+    cmd.add_argument(
+        '--run-indexing', action='store_true',
+        help='perform indexing',
+    )
+    cmd.add_argument(
+        '--remove-library-catalog', action='store_true',
+        #help='remove library catalog database file'
+        help=argparse.SUPPRESS,
+    )
+    cmd.add_argument(
+        '--remove-index-files', action='store_true',
+        help=argparse.SUPPRESS,
+    )
+    cmd.add_argument(
+        '--edit-config', action='store_true',
+        help='edit configuration file',
+    )
+    cmd.add_argument(
+        '--reinstall-dependencies', '-I', action='store_true',
+        help="reinstall your distro's dependencies",
+    )
+    cmd.add_argument(
+        '--backup', action='store_true',
+        #help='perform backup',
+        help=argparse.SUPPRESS,
+    )
+    cmd.add_argument(
+        '--restore', action='store_true',
+        #help='perform restore',
+        help=argparse.SUPPRESS,
+    )
+    cmd.add_argument(
+        '--set-appimage', action='store_true',
+        help=argparse.SUPPRESS,
+    )
+    cmd.add_argument(
+        '--get-winetricks', action='store_true',
+        help='download or update Winetricks',
+    )
+    cmd.add_argument(
+        '--run-winetricks', action='store_true',
+        help='start Winetricks window',
+    )
+    cmd.add_argument(
+        '--toggle-app-logging', action='store_true',
+        help='enable/disable app logs',
+    )
+    cmd.add_argument(
+        '--create-shortcuts', action='store_true',
+        help='[re-]create app shortcuts',
+    )
+    cmd.add_argument(
+        '--remove-install-dir', action='store_true',
+        help='delete the current installation folder',
+    )
+    cmd.add_argument(
+        '--dirlink', action='store_true',
+        #help='create directory link',
+        help=argparse.SUPPRESS,
+    )
+    cmd.add_argument(
+        '--make-skel', action='store_true',
+        #help='make a skeleton install only',
+        help=argparse.SUPPRESS,
+    )
+    cmd.add_argument(
+        '--check-resources', action='store_true',
+        #help='check resources'
+        help=argparse.SUPPRESS,
+    )
+    return parser
```

Comment:
I noticed in the other commits that you reduced the style for the GUI app to single lines rather than multilines.

While I tend to prefer single lines, is there a reason these are kept as multiline?

Reply:
I was having a hard time scanning back and forth, trying to confirm all of the parameters. Making in multiline made that easier. It also allowed adding comments mid-function. Plus, I tend to like to break up long lines, which we have with the inclusion of the "help" parameters. I'm not married to this formatting, though.